package.json

```diff
@@ -20,7 +20,8 @@
     "e2e:build-browsers": "docker build -f test/func/docker/Dockerfile -t html-reporter-browsers:0.0.1 --network host test/func/docker",
     "e2e:build-packages": "npm run --workspace=test/func/packages --if-present build",
     "e2e:generate-fixtures": "npm run --workspace=test/func/fixtures generate",
-    "e2e:launch-browsers": "docker run -it --rm --network=host --add-host=host.docker.internal:0.0.0.0 html-reporter-browsers:0.0.1",
+    "e2e:launch-browsers-local": "if which colima; then docker run -it --rm  --network=host html-reporter-browsers:0.0.1; else docker run -it --rm --network=host --add-host=host.docker.internal:0.0.0.0 html-reporter-browsers:0.0.1; fi",
+    "e2e:launch-browsers": "if which colima; then docker run -it --rm  --network=host yinfra/html-reporter-browsers; else docker run -it --rm --network=host --add-host=host.docker.internal:0.0.0.0 yinfra/html-reporter-browsers; fi",
```

Comment:
What's the difference between these two scripts? Only in the yinfra prefix? I'd keep a single script and always use yinfra/... image as I don't see any downsides of this approach.

Also, this may be simplified to something along the lines of:
```
docker run -it --rm --network=host $(which colima >/dev/null || echo --add-host=host.docker.internal:0.0.0.0) yinfra/html-reporter-browsers:0.0.1
```